common/locales/en/character.json

```diff
@@ -126,6 +126,7 @@
     "rogueText": "Rogues love to accumulate wealth, gaining more Gold than anyone else, and are adept at finding random items. Their iconic Stealth ability lets them duck the consequences of missed Dailies. Play a Rogue if you find strong motivation from Rewards and Achievements, striving for loot and badges!",
     "healerText": "Healers stand impervious against harm, and extend that protection to others. Missed Dailies and bad Habits don't faze them much, and they have ways to recover Health from failure. Play a Healer if you enjoy assisting others in your Party, or if the idea of cheating Death through hard work inspires you!",
     "optOut": "Opt Out",
+    "optOutMsg: "Opt Out",
```

Comment:
@luveluen there's a missing " before the semicolon `"optOutMsg: "Opt Out"` should be ""optOutMsg": "Opt Out""